docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,180 @@
+---
+description: Feature and improvement highlights for Grafana v9.5
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '9.5'
+  - release notes
+title: What's new in Grafana v9.5
+weight: -33
+---
+
+# What’s new in Grafana v9.5
+
+Welcome to Grafana 9.5! We're excited to share some major updates to Grafana's navigation, tons of usability improvements to Alerting, and some promising experiments to help you query your Prometheus metrics. Also, read on to learn about our continued migration from API keys to service accounts, as well as deprecation of plugins that use Angular and a field in the InfluxDB data source.
+
+For more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v9.5, check out our [Upgrade Guide]({{< relref "../../upgrade-guide/upgrade-v9.5/" >}}).
+
+<!-- Template below
+
+## Feature
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
+
+## Grafana's new navigation is GA
```

Comment:
@chri2547 - What do we think about this heading? It feels awkward to me. I've made a suggestion below.
Whatever we decide here, I'll be applying to line 59 since that was also a GA.
```suggestion
## Grafana's new navigation is generally available
```